game-core/src/main/java/games/strategy/engine/framework/startup/ui/posted/game/pbf/ForumPosterEditor.java

```diff
@@ -64,6 +75,48 @@
           SwingComponents.highlightLabelIfNotValid(viewModel.isForumUsernameValid(), usernameLabel);
           SwingComponents.highlightLabelIfNotValid(viewModel.isForumPasswordValid(), passwordLabel);
           testForum.setEnabled(viewModel.isTestForumPostButtonEnabled());
+          helpMessage.setText(
+              viewModel.isForumProviderTripleA()
```

Comment:
This is perfect :+1: 
The boolean is acting like a flag selecting between two values, looks good.